.github/workflows/test.yml

```diff
@@ -0,0 +1,453 @@
+# This file was auto-generated by .evergreen/make-github-actions
+# at 2021-08-21T15:14:55Z
+---
+name: Run Mongoid Tests
+'on':
+- push
+- pull_request
```

Comment:
Note: this will test every push and every pull request to all branches. For the PHP driver, I opted for the following configuration:
https://github.com/mongodb/mongo-php-library/blob/master/.github/workflows/tests.yml#L4..L11

This tests supported branches (the latest development branch and all stable release branches), both when changes are pushed and when a pull request targets one of those branches.

I don't think we necessarily need to limit this now, just pointing this out as we may find ourselves running more builds than we should, which I'd like to avoid in order to reduce load on open source build architecture.

Reply:
Let's run it all the time, so that contributors can see the results on their PRs. It's free--why not run it everywhere?